src/Veil.ts

```diff
@@ -26,8 +26,8 @@ export interface Market {
 
 export interface Order {
   uid: string;
-  longPrice: string;
-  longSide: "buy" | "sell";
+  price: string;
+  side: "buy" | "sell";
```

Comment:
Does coinbase pro use `side` as this key?

Reply:
yeah